src/components/Select/SelectBase/SelectBase.tsx

```diff
@@ -86,6 +88,12 @@ export function SelectBase<S>(props: Props<S>) {
             onSelectionChange(selectedItem);
         }
 
+        if (clearInputAfterSelection) {
+            setInputValue('');
+            setInputValueRecall('');
```

Comment:
`inputValueRecall` should not change, so that highlighting in list is still visible.

Reply:
Deleted it from here